legacy/src/main/resources/versions.properties

```diff
@@ -5,9 +5,9 @@ ksp=1.0.13
 nexus=1.3.0
 dokka=1.9.20
 jvmTarget=11
-serialization=1.6.3-SNAPSHOT
+serialization=1.6.3-SNAPSHOT!!
 datetime=0.5.0
 coroutines=1.8.0
 napier=2.7.1
-bouncycastle=1.77
+bouncycastle=1.77!!
```

Comment:
why?

Reply:
BC is explicitly set to strict by the conventions plugin, since we know from experience that we don't want version changes, because then what we have in the classpath might not be ABI-compatible with the compiled code.
That was just something I caught when setting this up